pynta/main.py

```diff
@@ -33,6 +38,7 @@ def __init__(self,path,rxns_file,surface_type,metal,label,launchpad_path=None,fw
                             }, },
         software_kwargs_gas=None,
         TS_opt_software_kwargs=None,
+        irc_mode=None, #choose irc mode: 'skip', 'relaxed', 'fixed'
```

Comment:
I think our default should be "fixed" at least until the v3.0 release to avoid a change in behavior.  Otherwise from a user perspective after adding these changes Pynta will simply stop calculating IRCs when before it calculated IRCs. When we release v3.0 we can change the defaults.

Reply:
fixed to irc_mode="fixed"